definitions/checks/foreman/check_puppet_capsules.rb

```diff
@@ -0,0 +1,38 @@
+module Checks
+  class CheckPuppetCapsules < ForemanMaintain::Check
+    metadata do
+      label :puppet_capsules
+      for_feature :foreman_database
+      description 'Check for Puppet capsules from the database'
+      manual_detection
+    end
+
+    def run
+      puppet_proxies = find_puppet_proxies.reject { |proxy| proxy['url'].include?(hostname) }
+      unless puppet_proxies.empty?
```

Comment:
Can we at least parse the URL and do a proper hostname comparisons? Hostnames are not simple strings: hostnames are case-insensitive. I'd be OK with ignoring IDN (International Domain Names, which can be recognized by starting with `xn--`)